src/main/java/com/shapesecurity/salvation/data/Policy.java

```diff
@@ -480,27 +493,34 @@ public boolean allowsScriptWithHash(@Nonnull HashAlgorithm algorithm, @Nonnull B
     }
 
     public boolean allowsUnsafeInlineScript() {
+        return this.hasUnsafeInlineScript() && !this.hasStrictDynamic();
+    }
+
+    public boolean hasUnsafeInlineScript() {
         ScriptSrcDirective scriptSrcDirective = this.getDirectiveByType(ScriptSrcDirective.class);
         if (scriptSrcDirective == null) {
-            return this.defaultsAllowUnsafeInline();
+            return this.defaultsHaveUnsafeInline();
         }
         return scriptSrcDirective.values().anyMatch(x -> x == KeywordSource.UnsafeInline);
     }
 
-    public boolean allowsUnsafeInlineStyle() {
+    public boolean hasUnsafeInlineStyle() {
         StyleSrcDirective styleSrcDirective = this.getDirectiveByType(StyleSrcDirective.class);
         if (styleSrcDirective == null) {
-            return this.defaultsAllowUnsafeInline();
+            return this.defaultsHaveUnsafeInline();
         }
         return styleSrcDirective.values().anyMatch(x -> x == KeywordSource.UnsafeInline);
     }
 
+    public boolean allowsUnsafeInlineStyle() {
+        return this.hasUnsafeInlineStyle();
```

Comment:
Why does this indirection exist? Just rename `hasUnsafeInlineStyle` to `allowsUnsafeInlineStyle`.